ambry-server/src/main/java/com/github/ambry/server/AmbryServerSecurityService.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.server;
+
+import com.github.ambry.commons.ServerMetrics;
+import com.github.ambry.config.ServerConfig;
+import com.github.ambry.rest.RestRequest;
+import com.github.ambry.rest.RestServiceErrorCode;
+import com.github.ambry.rest.RestServiceException;
+import com.github.ambry.router.Callback;
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * Default implementation of {@link ServerSecurityService} for Ambry that doesn't do any validations.
+ */
+public class AmbryServerSecurityService implements ServerSecurityService {
+  private boolean isOpen;
+  private final ServerConfig serverConfig;
+  private final ServerMetrics serverMetrics;
+
+  public AmbryServerSecurityService(ServerConfig serverConfig, ServerMetrics serverMetrics) {
+    this.serverConfig = serverConfig;
+    this.serverMetrics = serverMetrics;
+    this.isOpen = true;
+  }
+
+  @Override
+  public void validateConnection(ChannelHandlerContext ctx, Callback<Void> callback) {
+    Exception exception = null;
+    serverMetrics.securityServiceValidateConnectionRate.mark();
+    long startTimeMs = System.currentTimeMillis();
+    if (!isOpen) {
+      exception = new RestServiceException("ServerSecurityService is closed", RestServiceErrorCode.ServiceUnavailable);
+    } else if (ctx == null) {
+      throw new IllegalArgumentException("ctx is null");
+    }
+    serverMetrics.securityServiceValidateConnectionTimeInMs.update(System.currentTimeMillis() - startTimeMs);
```

Comment:
The timers should be moved to the calling method.  This is a no-op impl that doesn't need to be timed.

Reply:
done.